gradle-plugin/build.gradle

```diff
@@ -51,13 +53,24 @@ publishing {
     }
 }
 
+def getEnvVariable(String value) {
```

Comment:
to avoid confusion, the input variable could be called 'name' or 'envVariableName'

Reply:
Done